recipes/arpackpp/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.2" %}
+
+package:
+  name: arpackpp
+  version: {{ version }}
+
+source:
+  fn: arpackpp-{{ version }}.tar.gz
+  url: http://www.ime.unicamp.br/~chico/arpack++/arpack++.tar.gz
```

Comment:
I'm also seeing this [webpage]( http://reuter.mit.edu/software/arpackpatch/ ) for it, which points at this [repo]( https://github.com/m-reuter/arpackpp ).